src/nanotron/fp8/kernel.py

```diff
@@ -0,0 +1,68 @@
+import torch
+import transformer_engine as te  # noqa
```

Comment:
Should we add try except? Or add it as a dependency?

Reply:
Hmm, maybe we should use try-except. That way, we don't complicate things for people who just want to try out Nanotron (without having to install TE in order to try non-FP8 features...

Reply:
try except still missing no?

Reply:
these are the low-level files, we expect users to call FP8Tensor, FP8Linear, or Dtypes only (from nanotron.fp8) => we could skip it?